Source/WebCore/rendering/RenderTheme.cpp

```diff
@@ -223,7 +220,7 @@ void RenderTheme::adjustStyle(RenderStyle& style, const Element* element, const
     if (!supportsBoxShadow(style))
         style.setBoxShadow(nullptr);
 
-#if USE(NEW_THEME)
+#if !PLATFORM(IOS_FAMILY)
```

Comment:
A little sad to be writing something mechanical like this. I wish there was a name for this difference rather than just a platform.